### PR TITLE
feat(curriculum): add video compilation lab

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+```
+
+You should have three `section` elements below your first `p` element.
+
+```js
+assert.equal(document.querySelectorAll('section').length, 3);
+assert.equal(document.querySelectorAll('section')[0]?.previousElementSibling.tagName, 'P')
+```
+
+Each `section` element should contain an `h2` element to give a title to that section as its first child.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:57]: This test is always passing.

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+```js
+for (let e of $('section')) {
+    assert.equal(e.children[0]?.tagName, 'H2');
+    assert.isAbove(e.children[0]?.innerText.length, 0);
+}
+```
+
+Each `section` element should contain a `p` element to introduce the video content as its second child.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:66]: Always passing.

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+```js
+for (let e of $('section')) {
+    assert.equal(e.children[1]?.tagName, 'P');
+    assert.isAbove(e.children[1]?.innerText.length, 0);
+}
+```
+
+Each `section` element should contain an `iframe` element as its third child.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:75]: Always passing.

**Review by zairahira — CHANGES_REQUESTED** [review]

**Comment by Dario-DC** [comment]
Those three tests should work properly now @zairahira

**Review by zairahira — APPROVED** [review]

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -1713,6 +1709,10 @@
       "front-end-development-certification-exam": {
         "title": "Front End Development Certification Exam",
         "intro": ["", ""]
+      },
+      "lab-video-compilation-page": {
+        "title": "Video Compilation Page",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:1714]:
```suggestion
        "title": "Build a Video Compilation Page",
```

File: curriculum/challenges/_meta/lab-video-compilation-page/meta.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Video Compilation Page",
```
Comment (moT01) [inline curriculum/challenges/_meta/lab-video-compilation-page/meta.json:2]:
```suggestion
  "name": "Build a Video Compilation Page",
```

**Review by moT01 — COMMENTED** [review]
LGTM after conflicts fixed and title adjusted 👍

**Review by moT01 — APPROVED** [review]

**Pull request merged**